src/_includes/partials/page-footer/_page-footer.css

```diff
@@ -20,21 +21,28 @@
 
 .page-footer .desktop-columns {
   list-style-type: none;
-  max-width: var(--outer-width);
+  /* max-width: var(--outer-width); */
```

Comment:
Without this max width, what happens on larger form factors such as a 4k screen?

Reply:
Ah, crap. You're right. Commented it out to check something but forgot to uncomment it. Pushed it uncommented.